src/VisualStudio/Core/Test.Next/Services/SolutionServiceTests.cs

```diff
@@ -206,7 +207,33 @@ public async Task TestUpdateDocumentInfo()
         {
             var code = @"class Test { void Method() { } }";
 
-            await VerifySolutionUpdate(code, s => s.WithDocumentFolders(s.Projects.First().Documents.First().Id, new[] { "test" }));
+            using var workspace = TestWorkspace.CreateCSharp(code);
+
+            static Solution SetDocumentProperties(Solution solution, int version)
```

Comment:
Can we switch the version here to just accept a SourceCodeKind? The string + version then becoming string + version.ToString().

Reply:
i prefer this as a pattern used in the above test as well. There can be multiple properties that we need the version for.